src/SortChecker.cpp

```diff
@@ -210,6 +226,22 @@ class Visitor : public RecursiveASTVisitor<Visitor> {
     return true;
   }
 
+  bool isInterestingContainer(const RecordDecl *RD, unsigned CompareArg) const {
+    auto *SpecDecl = dyn_cast<ClassTemplateSpecializationDecl>(RD);
+    if (!SpecDecl)
+      return true;
```

Comment:
It seems it should be `false`, don't you think?

Reply:
Maybe this should be a `cast` instead because we do expect `RD` to always be a `SpecializationDecl`.

Reply:
You mean an `assert`?

Reply:
`cast` does `assert` internally.